### DISCUSSION TITLE
Extracting Spliced/unspliced ratio per cell

### DISCUSSION THREAD

**Discussion by JoshuaCumming** [theislab/scvelo#847] [Q&A]
Hi,

Is it possible to extract a spliced/unspliced ratio per cell from the Anndata object?

Thanks for any help!

**Comment by WeilerP** [comment]
There is no dedicated function, but if you have unspliced and spliced counts, you can simply calculate it yourself. Should (at least in case of dense arrays) be something along the lines of

```python
adata.layers['unspliced'].sum(axis=1) / (adata.layers['unspliced'] + adata.layers['spliced']).sum(axis=1)
```
